module-2/backend/week_two.md

```diff
@@ -5,16 +5,31 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. At a high level, what is ActiveRecord? What does it do/allow you to do?
+  - ActiveRecord is an Object Relational Mapper (ORM), which is a relational database wrapper that helps the database and ruby classes talk to each other. Each table in the database represents a ruby class, each row in the table represents an instance of the class, and each column of the table represents an attribute of the class. It allows you to run SQL queries through shorthand ActiveRecord methods.
 2. What kind of methods are `belongs_to`, and `has_many`? (i.e. class or instance) Give an example.
+  - 'belongs_to' and 'has_many' are class methods. Other examples of class methods are 'all' and 'first'.
 3. What do they allow you to do?
+  - 'belongs_to' and 'has_many' allow you to set up associations between two tables in the database.
```

Comment:
This is true to some degree, but I think it's more accurate to say that it allows you to *use* the relationships set up in the DB. The big difference being that you won't be able to use `belongs_to` or `has_many` without a foreign key already existing in your database. I mention it because sometimes students set up `belongs_to` and/or `has_many` and then can't figure out why their relationships aren't working.